php/commands/super-admin.php

```diff
@@ -27,9 +27,10 @@ public function _list( $_, $assoc_args ) {
 	 * <user>...
 	 * : One or more user IDs, user emails, or user logins.
 	 */
-	public function add( $args, $_ ) {
+	public function add() {
```

Comment:
One potential problem is that if something calls this method directly, `$args` don't get considered in `get_positional_args()`